src/comment.rs

```diff
@@ -0,0 +1,221 @@
+const PREFIX: &str = "dotenv-linter";
+const ON: &str = "on";
+const OFF: &str = "off";
+
+#[derive(Debug, PartialEq)]
+pub struct Comment<'a> {
+    disable: bool,
+    pub checks: Vec<&'a str>,
+}
+
+pub fn parse(s: &str) -> Option<Comment> {
+    // A comment without the # character
+    let line_comment = s[1..].trim();
+
+    if !line_comment.starts_with(PREFIX) {
+        return None;
+    }
+
+    // Split the comment into 2 parts - ["dotenv-linter", "off UnorderedKey, DuplicatedKey"]
+    let parts: Vec<&str> = line_comment.split(':').collect();
```

Comment:
Is it better to use `strip_prefix` ? 
The split seems excess because we are not using left part.

```suggestion
    let right_parts: &str = line_comment.strip_prefix(PREFIX + ':')?;
```

Reply:
Yes, you're right. Thanks for advice! 🙏